de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/SarosSessionManager.java

```diff
@@ -589,9 +591,9 @@ public void addResourcesToSession(
                     TransferType type = TransferType.valueOf(currentSession
                         .getUserProperties(user).getString(
                             ProjectNegotiationTypeHook.KEY_TYPE));
-                    AbstractOutgoingProjectNegotiation negotiation = negotiationFactory
-                        .newOutgoingProjectNegotiation(user.getJID(), type,
-                            projectsToShare, this, currentSession);
+                    AbstractOutgoingReferencePointNegotiation negotiation = negotiationFactory
+                        .newOutgoingReferencePointNegotiation(user.getJID(), type,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "getJID"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMdFDZywvPe4LUfP&open=AWZfKMdFDZywvPe4LUfP)

[ISSUE_KEY]: AWZfKMdFDZywvPe4LUfP